lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -2590,6 +2590,21 @@ def MigrateInstance(self, cluster_name, instance, target, live_migration):
     self._SetInstanceMigrationCapabilities(instance)
     self.qmp.StartMigration(target, port)
 
+  @_with_qmp
+  def StartPostcopy(self, instance):
+    """Switch a migration from precopy to postcopy mode.
+
+    Requires that an instance is currently migrating, and that the
+    postcopy-ram (x-postcopy-ram on QEMU version 2.5 and below)
+    migration capability is enabled in the instance's hypervisor
+    parameters.
+
+    @type instance: L{objects.Instance}
+    @param instance: The instance being migrated.
+
+    """
+    self.qmp.StartPostcopyMigration()
```

Comment:
Have we checked if this fails when issued against a completed migration? This is important because some time will have passed from the moment we issued the `migration_info` command and regular migration may have succeeded.